lib/rules/no-test-return-value.js

```
@@ -80,6 +80,10 @@ module.exports = {
               {
                 messageId: 'suggest',
                 fix(fixer) {
+                  if (!node.range || !node.argument || !node.argument.range) {
+                    // The if statement is just to make TypeScript happy.
```

In that case, would it be better to throw an error instead?

---

Shouldn't matter too much but I updated to an exception.